app/models/user.rb

```diff
@@ -0,0 +1,32 @@
+class User < ActiveRecord::Base
+  attr_accessor :password
+  before_save :encrypt_password
+  before_create :confirmation_token
+
+  validates :password, confirmation: true
```

Comment:
This validation only runs if `password_confirmation` is provided. In your controller above [this line](https://github.com/npauzenga/Blocitoff/pull/1/files#diff-4e05ad0d64e6100656b63ad1e78f32c5R32) you are filtering out `password_confirmation`, so it will never be set. It's very common now to see registration flows without requiring users to confirm their password. It's completely up to you :smile:. If you want to use password confirmation then whitelist it in your controller, if not then you can remove the validation.